### DISCUSSION TITLE
Assumption that users will be reading books?

### DISCUSSION THREAD

**Discussion by spudthebud** [edrlab/thorium-reader#1747] [Q&A]
I was stepping through the process of downloading an academic journal article as an ePub and opening it in Thorium. I was using the JAWS screen reader with an eye mask on, so I didn't have access to the visual interface.  Naturally, I was listening greatly.  I found it difficult to do the task. While it could be because of my skill level, I also wonder if Thorium has made a design assumption that people will only be reading books. I noticed a lot of labels that use "Books".  I wonder if that could mislead others who are using Thorium to read non-book material?

**Comment by danielweck** [comment]
Thank you for your feedback. What alternative wording would you suggest? Given that EPUB files can implement various different types of publications, I suppose we could use a more neutral term than "book"?

**Comment by spudthebud** [comment]
"Reading material" is accurate but long. "Readings" is shorter, and could convey things like books, articles, reports, and documents.  I wonder if there's a way to get some user feedback?

**Comment by MyDK** [comment]
we use 'book' as this term covers a lot of different materials, genres and formats and publishers are comfortable with this term. But 'content' might more accurate, also covering dynamic media formats, right? (Hope it is okay, I butt in :)

**Comment by spudthebud** [comment]
@MyDK , as mentioned the user path I was going down was downloading an ePub of an article that was published by a major academic journal. The next step was to open it in Thorium and then read the article.  For the step of navigating the interface to read it, I found the word "book" confusing, because I was not doing the task of reading a book.
